BootstrapScriptWithToken/bootstrap.py

```diff
@@ -206,24 +207,31 @@ def tryImageUpgrade( e ):
 # script with python3. In case we cannot recover, the script will require "eosUrl" to
 # perform an upgrade before it can proceed.
 try:
-   from SysdbHelperUtils import SysdbPathHelper
-   import requests
    import Cell
+   import requests
+   from SysdbHelperUtils import SysdbPathHelper
 except ImportError as e:
    if sys.version_info < (3,) and os.path.exists( '/usr/bin/python3' ):
       os.execl( '/usr/bin/python3', 'python3', os.path.abspath(__file__ ) )
    else:
       log("Python3 not found. Attempting EOS version upgrade")
       tryImageUpgrade( e )
 
-if sys.version_info >= (3,):
+try:
+   # This import will fail when running this script in python2 environment
+   # starting EOS 4.30.1
+   # We can't use the conditional logic here to import the libraries as pylint will
```

Comment:
conditional logic does say what we used to do. You can remove the comments from line 223-224. Line 221-222 should be enough.

Reply:
Done